README.md

```diff
@@ -59,33 +59,24 @@ Setting `noReset` to `true` will leave the simulator running at the end of a tes
 
 The `appium-xcuitest-driver` has provisional support for iOS real devices. Not all functionality is currently supported.
 
-The main issue with testing on real devices is that the app under test must be built with the same provisioning profile as the WebDriverAgent is built. At the moment, the command being used to build the WebDriver agent is (with adjustments for paths):
+WebDriverAgent needs to be built with development team and provisioning profile installed on device.
 
-```
-xcodebuild \
-  -project' <path-to-WebDriverAgent>/WebDriverAgent.xcodeproj \
-  -scheme WebDriverAgentRunner \
-  -sdk iphoneos \
-  -destination platform=iOS,id=<UDID> \
-  CODE_SIGN_IDENTITY=iPhone Developer \
-  CODE_SIGNING_REQUIRED=YES \
-  test
-```
+* Open terminal go to `/appium-xcuitest-driver/WebDriverAgent` and then run `sh ./Script/build.sh`
 
-Therefore it is recommended that the app under test be built with the same command, making sure it is built for the same device and, also importantly, for debugging:
+* Open `WebDriverAgent.xcodeproj`. Select your `development team`. This should also auto select `Signing Ceritificate`. This should look like as shown below.
 
-```
-xcodebuild \
-  -project ./<APP>.xcodeproj \
-  -sdk iphoneos \
-  -destination platform=iOS,id=<UDID> \
-  CODE_SIGN_IDENTITY='iPhone Developer' \
-  CODE_SIGNING_REQUIRED=YES \
-  -configuration Debug \
-  clean build
-```
+  ![alt WebDriverAgent in Xcode project](https://cloud.githubusercontent.com/assets/12143988/18771980/2dc4f412-80f8-11e6-9ad6-c6883dbf6a03.png)
+
+* Build `WebDriverAgent` once to verify all above steps worked. 
+  ```
+    xcodebuild -project WebDriverAgent.xcodeproj -scheme WebDriverAgentRunner -destination 'id=<udid>' test
+
+  ```
+  Last line on build ouput above command should be `Listening on USB`. Then you are all set!
+
+Internally it also expects `deviceconsole` to be installed. (You can install `npm install deviceconsole`. A compiled binary named `deviceconsole` will be added to `./node_modules/deviceconsole/`.)
 
-Internally it also expects `idevicesyslog` to be installed (see installation instructions for [libimobiledevice](http://www.libimobiledevice.org/)).
+You need to either have `deviceconsole` in your `PATH`, or specify its location using the `realDeviceLogger` capability.
```

Comment:
It might be better to say "...or specify the absolute path to its location using the `realDeviceLogger` capability"